core/storage/classroom/gae_models.py

```diff
@@ -55,6 +67,14 @@ class ClassroomModel(base_models.BaseModel):
     # prerequisite topic IDs as value.
     topic_id_to_prerequisite_topic_ids = datastore_services.JsonProperty(
         indexed=False, required=False)
+    # Whether this classroom is published or not.
+    # False if classroom is hidden, True if published.
+    is_published = datastore_services.BooleanProperty(
+        indexed=True, required=True, default=False)
+    # The thumbnail data of the classroom.
+    thumbnail = datastore_services.JsonProperty(indexed=False)
```

Comment:
@AkashPaloju When you write "thumbnail", this sounds like the thumbnail itself, e.g. an image. thumbnail_data might be a better name, and ditto for banner_data below.

This probably needs to be fixed throughout.

Reply:
Done.